Logger context (#794)
yiisoft/db: docs/en/connection/logger.md

```

For other DBMS refer to ["Create connecton"](/docs/en/README.md#create-connection) section.

## Advanced usage of Logger

> If you needed re-define logger messages or increase(decrease) level of log:

Reviewer: ```suggestion
If you need to redefine logger messages or increase/decrease logging level:

```

> 1. Create custom logger class
> 2. Use context for detect type of message in method "log"

Reviewer: ```suggestion
1. Create a custom logger class
2. Use the context to detect type of the message in the "log" method
```
